Cartfile

```diff
@@ -1,8 +1,8 @@
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.3.0
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 22.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 24.0
```

Comment:
Maybe we want v25.0 now?

Reply:
Actually, yes, you are right. In v25 NN [renamed Incident type](https://github.com/mapbox/mapbox-navigation-native/pull/1787) JSON key which affects the feature.